fix: get downsample back in the game

drdelambre opened the pull request:
for some reason it wasn't working so good, but it looked like it should have worked

hercules-influx commented:
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 58 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=1153)

Asalem1 reviewed (approved):
Should we get a test in on this so that it doesn't break in the future?

drdelambre commented:
i would agree if i knew that this was the correct behavior. i still have nightmares about that time that watts renamed notebooks everywhere into flows, then we changed the name back to notebooks. and technically... it _was_ working perfectly fine, there was just a choice to put a default value on the view layer that.. changed it's expression when we cleaned up the aggregate window work on the visualization layer. idk.. conflicted
